tests/integ/test_git.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import numpy
+import tempfile
+
+from tests.integ import lock as lock
+from sagemaker.mxnet.estimator import MXNet
+from sagemaker.pytorch.estimator import PyTorch
+from tests.integ import DATA_DIR, PYTHON_VERSION
+
+GIT_REPO = 'https://github.com/aws/sagemaker-python-sdk.git'
+BRANCH = 'test-branch-git-config'
+COMMIT = '329bfcf884482002c05ff7f44f62599ebc9f445a'
+LOCK_PATH = os.path.join(tempfile.gettempdir(), 'sagemaker_test_git_lock')
```

Comment:
add a comment here about why this lock is needed - you can copy this one: https://github.com/aws/sagemaker-python-sdk/blob/master/tests/integ/test_local_mode.py#L31

Reply:
The lock here prevents the race condition where the same port on local machine is used for deploying models in several tests at the same time

Reply:
haha my bad - I meant add a comment in the code